### PR TITLE
Set an RPATH on all libraries requiring fortran in binary-dist

### PR DESCRIPTION
**Pull request by Keno**
@tkelman see if this is what you had in mind

### CODE REVIEW

File: contrib/fixup-libgfortran.sh [hunk 1]
```diff
@@ -93,3 +93,12 @@ if [ "$UNAME" = "Darwin" ]; then
         done
     done
 fi
+
+if [ "$UNAME" = "Linux" ]; then
+    cd $private_libdir
+    for file in openlibm quadmath gfortran openblas arpack lapack openspecfun; do
+        for dylib in $(ls lib$file*.so* 2>/dev/null); do
+            patchelf --set-rpath \$ORIGIN $dylib
```
Comment (tkelman) [inline contrib/fixup-libgfortran.sh]: do we have the `PATCHELF` exported in a way that would propagate? someone might have it somewhere else, but I guess that's a second-order concern here

**Comment by tkelman** [comment]
This should help, thanks

**Comment by Keno** [comment]
Somebody could override it, I suppose, but would such a somebody be making binary-dists, who knows. I don't think I particularly care.

File: contrib/fixup-libgfortran.sh (hunk 1 above)
Comment (vtjnash) [inline contrib/fixup-libgfortran.sh]: yes, we build patchelf ourself and call it `$(PATCHELF)`

Comment (Keno) [inline contrib/fixup-libgfortran.sh]: That variable is only set in deps/, if you feel like rearranging things and threading it through, be my guest, but this code catches patchelf globally or in usr/tools, so I don't really see the need to do the extra work to make it work with an override `PATCHELF`.

**Pull request merged**